src/cascadia/TerminalSettingsModel/dll/pch.h

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// pch.h
+// Header for platform projection include files
+//
+
+#pragma once
+
+// This file can be empty - the pch.h in TerminalSettingsModel/lib does the heavy lifting
+// of including all the headers we need. As this project is just a dll wrapper,

```

Comment:
are these craps still necessary

Reply:
did we copy it because we copied it, or did we copy it because it will break otherwise

Reply:
Copied because it was copied. Just removed it. Wasn't needed at all.